config/app.default.php

```diff
@@ -62,7 +63,7 @@
      *   You should treat it as extremely sensitive data.
      */
     'Security' => [
-        'salt' => '__SALT__',
+        'salt' => env('SECURITY_SALT', '__SALT__'),
```

Comment:
Am against prefixing these constants. They become insanely long in real scenarios.